ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -1003,6 +1003,46 @@ AHU_DXSC_ECON_EFSS_SFSS:
   - EFSS
   - SFSS
 
+AHU_DXZC_ECONZ_EFSS_DFSS_DAIDPC:
+  guid: "fab797a6-3607-410d-af07-22ad1fec322f"
+  description: "Single zone AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - DXZC
+  - ECONZ
+  - EFSS
+  - DFSS
+  - DAIDPC
+
+AHU_DX2ZC_ECONZ_EFSS_DFSS_DAIDPC:
+  guid: "0d7f5a82-c804-44dc-87ac-5b390530ca36"
+  description: "Single zone AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - DX2ZC
+  - ECONZ
+  - EFSS
+  - DFSS
+  - DAIDPC
+
+AHU_CONWSC_ECON_EDBPC_EFSS_EFVSC_SFSS_SFVSC_SSPC_SARC:
+  guid: "b9c13f51-6c80-4507-948c-d0e82927c218"
+  description: "Multi-zone AHU with changeover heating and cooling valve, supply air static pressure control, and building static pressure control."
```

Comment:
Please change the description to be `...with thermally conditioned water valve...` as the entity does not actually contain the changeover valves, these are in the water system itself.